### PR TITLE
Conditional MDAnalysis content

### PR DESCRIPTION
**Pull request by ianmkenney**
In this PR I'm looking to only include the official MDA assets when the "mdaofficial" option is set to true.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -107,7 +107,7 @@
     "base_url": "https://minium.com.au/mdanalysis-sphinx-theme/",
     "repo_url": "https://github.com/lilyminium/mdanalysis-sphinx-theme/",
     "repo_name": "mdanalysis-sphinx-theme",
-
+    "mdaofficial": True,
```
Comment (lilyminium) [inline docs/conf.py]: Just stylistically -- would you be ok with an underscore in between (e.g. `mda_official`, or `mdanalysis_official`) to fit with the other options in the theme?

Comment (ianmkenney) [reply docs/conf.py]: Absolutely, I'm going to make a lot more changes tomorrow

File: setup.py [hunk 2]
```diff
@@ -1,8 +1,47 @@
...
+    static_dir = Path('.') / 'mdanalysis_sphinx_theme' / 'static'
+    branding_dir = Path('.') / 'branding'
+
+    # outdir for logos and icons
+    static_logos = static_dir / 'logo'
+
+    # main logo to appear in nav bar
+    mda_logo: Path = branding_dir / 'logos' /\
+                                    'rastered' /\
+                                    'mdanalysis-logo_bgwhite@600ppi.png'
```
Comment (lilyminium) [inline setup.py]:
```suggestion
    mda_logo_base_file = "mdanalysis-logo_bgwhite@600ppi.png"
    mda_logo: Path = branding_dir / 'logos' / 'rastered' / mda_logo_base_file
```

Could we maybe get this path onto one line with the above workaround, or maybe encapsulate it in parentheses? In general PEP8 prefers shorter lines, or wrapping code in parentheses, over backslash-style line continuation.

File: setup.py [hunk 3]
```diff
@@ -1,8 +1,47 @@
...
+
+    mda_favicon = branding_dir / 'logos' /\
+                                 'icons' /\
+                                 'mdanalysis-logo.ico'
```
Comment (lilyminium) [inline setup.py]: Here as well -- the full thing on one line is only 73 characters, could we use that instead?

**Review by lilyminium — CHANGES_REQUESTED** [review]
After some fiddling with submodules I got this to build and it looks great, but I think RTD may need [additional settings in its config file](https://docs.readthedocs.io/en/stable/config-file/v2.html#submodules), probably like the below:

```
submodules:
  include:
    - branding
```

Could you please try adding that so we can get RTD to build example docs?

The GitHub Actions deployment is also failing -- I think the checkout action might also need additional configuration, but I assume it'll fail on the push to `gh-pages` anyway since this is a fork.

**Comment by ianmkenney** [comment]
> The GitHub Actions deployment is also failing -- I think the checkout action might also need additional configuration, but I assume it'll fail on the push to `gh-pages` anyway since this is a fork.

Hopefully it will at least build now since I made some changes to the workflow to include submodules

**Comment by ianmkenney** [comment]
🥳

**Review by lilyminium — APPROVED** [review]
Everything LGTM -- thank you for your work putting this together @ianmkenney, looks fantastic!

**Pull request merged**